python/delta/tables.py

```diff
@@ -594,6 +594,27 @@ def addFeatureSupport(self, featureName: str) -> None:
         DeltaTable._verify_type_str(featureName, "featureName")
         self._jdt.addFeatureSupport(featureName)
 
+    @since(4.0)  # type: ignore[arg-type]
+    def dropFeatureSupport(self, featureName: str) -> None:
+        """
+        Modify the protocol to drop a supported feature. The operation always normalizes the
+        resulting protocol. Protocol normalization is the process of converting a table features
+        protocol to the weakest possible form. This primarily refers to converting a table features
+        protocol to a legacy protocol. A Table Features protocol can be represented with the legacy
```

Comment:
```suggestion
        protocol to a legacy protocol. A Table Feature protocol can be represented with the legacy
```

Let's use "Table Feature" over "table feature" in all descriptions